src/Avalonia.Themes.Fluent/Controls/PathIcon.xaml

```diff
@@ -8,22 +8,18 @@
       <PathIcon Data="{StaticResource settings_regular}" />
     </StackPanel>
   </Design.PreviewWith>
-  <Style Selector="PathIcon">
-    <Setter Property="Foreground" Value="{DynamicResource TextControlForeground}" />
```

Comment:
I removed the `Foreground` setter here because it's really useful if `PathIcon` inherits the current foreground color instead of resetting it. Is that OK @maxkatz6 ?

Reply:
Yes, definitely.